tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
this is modified from pilot's right ? can you add a link to the original one in the header (and in a pilot PR so people are aware they need to modify both some times) ?
with a TODO: unify/have 1